### PR TITLE
(CONT-1134) PDK 3.0.0 Release Prep

### PR DESCRIPTION
**Pull request by chelnak**
## Summary
Provide a detailed description of all the changes present in this pull request.

## Additional Context
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process behind the implementation.

## Related Issues (if any)
Mention any related issues or pull requests.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ To use, execute `pdk console` from inside your module directory.  You can also s
 
 Example (from within a module):
 
-* `pdk console --puppet-version=5`
+* `pdk console --puppet-versio 7`
```
Comment (jordanbreen28) [inline README.md:119]:
```suggestion
* `pdk console --puppet-version=7`
```

File: docs/pdk_install.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ By default, PDK installs to the following locations:
 -   On Windows systems: `C:\Program Files\Puppet Labs\DevelopmentKit`
 
 
-PDK uses Puppet 6 and 7.
+PDK uses the latest versions pf Puppet 7 and 8 available at the time of release.
```
Comment (jordanbreen28) [inline docs/pdk_install.md:13]:
```suggestion
PDK uses the latest versions of Puppet 7 and 8 available at the time of release.
```

File: docs/pdk_upgrading.md [hunk 3]
```diff
@@ -7,42 +7,108 @@ Upgrade PDK using the same method you used to originally install it. See the PDK
...
+
+```
+  optional:
+    ":development":
+    - gem: racc
+      version: '~> 1.4.0'#
+      condition: if Gem::Requirement.create(['>= 2.7.0', '< 3.0.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))
+```
+
+Note: If you opt for manually adding the version requirement to your gem file, you will need
```
Comment (jordanbreen28) [inline docs/pdk_upgrading.md:90]: unfinished note

File: docs/pdk_upgrading.md [hunk 4]
```diff
@@ -7,42 +7,108 @@ Upgrade PDK using the same method you used to originally install it. See the PDK
...
+
+##### Resolution
+
+To resolve this issue you can either:
+
+* Remove github_changelog_generator from your `.sync.yml` and use another solution for changelog generation.
+
+* Set the version requirement to 1.15.2 in your `sync.yml`.
+
+After completing the step of your choice, you will need to run `pdk update` to ensure that you
```
Comment (jordanbreen28) [inline docs/pdk_upgrading.md:113]:
```suggestion
After completing the step of your choice, you will need to run `pdk update` to ensure that your
```

File: docs/pdk_upgrading.md [hunk 5]
```diff
@@ -7,42 +7,108 @@ Upgrade PDK using the same method you used to originally install it. See the PDK
...
+Gemfile is updated appropriately.
```
Comment (jordanbreen28) [inline docs/pdk_upgrading.md:114]: Just a suggestion
```suggestion
Gemfile is updated accordingly.
```

File: README.md [hunk 6]
```diff
@@ -116,7 +116,7 @@ To use, execute `pdk console` from inside your module directory.  You can also s
 
 Example (from within a module):
 
-* `pdk console --puppet-version=5`
+* `pdk console --puppet-version=7`
```
Comment (davidsandilands) [inline README.md:119]: I am assuming debugger isn't tested with Puppet 8 so this makes sense being 7 ?

**Pull request merged**